benches/bench_multiprocess/src/main.rs

```diff
@@ -129,7 +129,7 @@ fn blake2b_hash_many() -> u128 {
     }
     let params = blake2b_simd::Params::new();
     bench(|| {
-        let mut jobs = arrayvec::ArrayVec::<[_; blake2b_simd::many::MAX_DEGREE]>::new();
+        let mut jobs = arrayvec::ArrayVec::<_, { blake2b_simd::many::MAX_DEGREE }>::new();
```

Comment:
Interesting, I just confirmed that you're right, that braces are needed here, but I'm not sure I understand why. Do you know why?

Reply:
The braces mark it as an expression to evaluate... a very simple one, obviously. Eventually such largely redundant bracing is intended to be optional but for now the parser is very conservative.